### PR TITLE
Prioritize styled-components #2171, upgrade react, material#2170

### LINKED ISSUE #2171: Fix Priority of styled-components in Material UI
**Actual Behaviour**

Both styled-components and JSS inject their styles at the bottom of the <head>. Styles of styled-components are not applied to material ui.

**Expected Behaviour**

To prioritize the styled-components, we have to use `StyleProvider`

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2171, #2170

Changes:
1. Fix Priority of styled-components in Material UI
2. Upgrade React to: `16.8.6`, Material UI Version to `4.0.2`
3. Remove `&&` class bumps used earlier
4. Integrate styled component in ToTopButton.

Demo Link: https://pr-2172-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/cms/SkillCreator/SkillCreator.js [hunk 1]
```diff
@@ -151,41 +151,35 @@ const CenterDiv = styled.div`
...
-    color: #9e9e9e;
-    display: ${mobileView ? 'block' : 'inline-bock'};
-  }
+  position: relative;
+  float: right;
+  height: 1.25rem;
+  width: 1.25rem;
+  cursor: pointer;
+  color: #9e9e9e;
+  display: ${mobileView ? 'block' : 'inline-bock'};
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillCreator.js]: Use media queries

File: src/components/cms/SkillCreator/SkillCreator.js [hunk 2]
```diff
@@ -151,41 +151,35 @@ const CenterDiv = styled.div`
...
-    display: ${isMobileView(769) ? 'none' : 'inherit'};
-  }
+  position: absolute;
+  left: 0.25rem;
+  top: 0.25rem;
+  width: 2.188rem;
+  height: 2.188rem;
+  color: white;
+  cursor: pointer;
+  display: ${isMobileView(769) ? 'none' : 'inherit'};
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillCreator.js]: Use media queries

File: src/components/cms/SkillCreator/SkillCreator.js [hunk 3]
```diff
@@ -151,41 +151,35 @@ const CenterDiv = styled.div`
...
-    display: ${isMobileView(769) ? 'none' : 'inherit'};
-  }
+  position: relative;
+  left: -1.875rem;
+  top: -1.25rem;
+  width: 2.188rem;
+  height: 2.188rem;
+  color: #9e9e9e;
+  cursor: pointer;
+  display: ${isMobileView(769) ? 'none' : 'inherit'};
```
Comment (akshatnitd) [inline src/components/cms/SkillCreator/SkillCreator.js]: Same here

**Comment by fragm3** [comment]
@akshatnitd I have fixed the links, should I make the above changes in a new PR, the skills.susi.ai needs refactoring for mobile view.

**Pull request merged**